### ISSUE TITLE
Bug audio Browser Edge 

### ISSUE THREAD

**Issue by Seby68** [FrostCo/AdvancedProfanityFilter#501]
Hi Richard, I wanted to report a bug to you, on Edge browser, the function in caption/Subtitles, hide all and show only filtered, doesn't work.

**Comment by richardfrost** [comment]
Which site(s) @Seby68?

**Comment by Seby68** [comment]
Richard, I've noticed that it doesn't work when the Mute Method is set to video mute, but on the Browser tab, it works. I don't know if this is normal or just a bug. Thanks and bye.

**Comment by richardfrost** [comment]
@Seby68 is it working like that on both mediaset.it and Netflix? I tried testing it on Netflix (USA) and it seemed to work in Edge for both the Browser tab and video mute methods.

Just to make sure I'm looking at the right thing, is muting working fine on both sites, its just not correctly hiding the captions/subtitles when it should?

**Comment by Seby68** [comment]
Yes right, it doesn't hide subtitles when it should. Hello Richard and always many thanks, for your seriousness.
